### PR TITLE
Merge `main` into `development`

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Merging `main` into `development`, which is mostly the contents of #340 but also includes the `2024.08.01` CHANGELOG. I need to go back into development and fix one thing after this; I added a comment about that.

### CODE REVIEW

File: docs/download_files.md [hunk 1]
```diff
@@ -164,8 +164,10 @@ Metadata for all samples on the Portal is available to download separately from
 Each project page has an option to download metadata for all of its samples as a single zip file containing the `metadata.tsv` file and a `README.md` file.
 Project-specific metadata will contain all columns listed in [the above table](#metadata) and any additional project-specific columns, such as treatment or outcome.
 
+<!--
 Additionally, a single TSV file containing the metadata for all samples from all projects on the Portal is available for download.
 The Portal-wide metadata will contain all columns listed in [the above table](#metadata).
+-->
```
Comment (jaclyn-taroni) [inline docs/download_files.md:170]: I will need to go back into `development` and uncomment this.

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -12,10 +12,23 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
-## PLACEHOLDER FOR PORTAL WIDE METADATA
-
-* Metadata for all samples from all projects on the Portal can now be downloaded in a single tab-separated values file.
-* For more information on what to expect in the metadata file, see the {ref}`metadata section of the Downloadable files page <download_files:metadata>`.
```
Comment (jaclyn-taroni) [inline docs/CHANGELOG.md:18]: I also need to add this back to `development`.

Comment (jashapiro) [reply docs/CHANGELOG.md:18]: Seems reasonable to do this in a separate PR, rather than dealing with a messy branch just for this merge.

**Review by jashapiro — APPROVED** [review]
With the noted comments expected as a followup PR, this seems fine.

**Pull request merged**